src/highlight-effekt.js

```diff
@@ -66,40 +66,9 @@ hljs.registerLanguage("effekt", function highlightEffekt(hljs) {
     relevance: 0
   };
 
-  var CLASS = {
-    className: 'class',
-    beginKeywords: 'class object trait type',
-    end: /[:={\[\n;]/,
```

Comment:
I don't think we really used this anywhere except for `type` declarations so I think I merged the important parts with `METHOD` to make `DEFINITION`.